src/utils/date.ts

```diff
@@ -91,7 +103,15 @@ export function formatToClockOrRelativeMinutes(
 
   return secondsToMinutesShort(diff);
 }
-
+export function formatLocaleTime(date: Date, language?: Language) {
```

Comment:
Does it exist something in date-fn that can handle this?

Reply:
Something.. Perhaps! Haven't looked into it yet, the date functions are still on the todo list